webusb/idlharness.html

```diff
@@ -0,0 +1,216 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset=utf-8>
+    <title>WebUSB IDL test</title>
+    <link rel="help" href="https://wicg.github.io/webusb/">
+    <script src=/resources/testharness.js></script>
+    <script src=/resources/testharnessreport.js></script>
+    <script src=/resources/WebIDLParser.js></script>
+    <script src=/resources/idlharness.js></script>
+  </head>
+  <body>
+    <script type="text/plain" id="untested">
+      interface Event {};
+      interface EventHandler {};
+      interface EventTarget {};
+      interface Navigator {};
+    </script>
```

Comment:
`idlharness.js` currently doesn't test dictionary definitions but you're right it's good to add.